Update Snapshot docs for SLM (#1900)
elastic/cloud-on-k8s: docs/snapshots.asciidoc

    secureSettings:
    - secretName: gcs-credentials
----

> If you did not follow the instructions above and named your GCS credentials file differently, you can still map it to the expected name now. See link:k8s-es-secure-settings.html[Secure Settings] for details.

Reviewer: ```suggestion
If you did not follow the instructions above and named your GCS credentials file differently, you can still map it to the expected name now. See <<{p}-es-secure-settings,Secure Settings>> for details.
```

[float]
[id="{p}-setup-cronjob"]
==== Periodic snapshots with Snapshot Lifecycle Management

added[7.4.0]

> You can use the https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-lifecycle-management-api.html[Snapshot Lifecycle Management APIs] to manage policies for the time and frequency of automatic snapshots.

Reviewer: ```suggestion
You can use the https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-lifecycle-management-api.html[snapshot lifecycle management APIs] to manage policies for the time and frequency of automatic snapshots.
```

Not sure whether we are supposed to use title case for features. It looks a bit odd to me.

The https://www.elastic.co/guide/en/kibana/current/snapshot-repositories.html[Snapshot and Restore UI] allows you to manage these policies directly in Kibana as well.

==== Periodic snapshots with a CronJob

> If you are running older versions of Elasticsearch, that don't have Snapshot Lifecycle Management, you can still set up a simple CronJob to take a snapshot every day.

Reviewer: ```suggestion
If you are running older versions of Elasticsearch without the snapshot lifecycle management feature, you can still set up a simple CronJob to take a snapshot every day.
```

> If you are running older versions of Elasticsearch, that don't have Snapshot Lifecycle Management,  you can still set up a simple CronJob to take a snapshot every day.

Reviewer 2: Nit: double space, `s|,  you|, you|`
